database/migrations/2023_12_20_063600_create_authentications_table.php

```diff
@@ -11,22 +11,20 @@
      */
     public function up(): void
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('authentications', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
-            $table->rememberToken();
+            $table->string('token')->unique();
+            $table->string('email');
+            $table->integer('status');
+            $table->timestamp('expiration_at')->default(now()->addMinutes(15));
```

Comment:
```suggestion
            $table->timestamp('expired_at')->default(now()->addMinutes(15));
```
〇〇_atというカラム名にする場合、大抵の場合は過去分詞形にします。
expireも感覚的には未来を想像されると思いますが、後から確認すると過去の日時の場合が多いので、過去分詞形にする方が読みやすいです。

Reply:
ご指摘、ご説明ありがとうございます。
有効期限のカラム名をexpired_atへと変更致しました。
ご確認よろしくお願い致します。
database/migrations/2023_12_20_063600_create_authentications_table.php